### ISSUE TITLE
python tools/converter.py run no  responses

### ISSUE THREAD

**Issue by raninbowlalala** [XiaoMi/mace#73]
I run "python tools/converter.py build --config=mace/models/deeplab-v3-plus/deeplab-v3-plus-mobilenet-v2.yml" successfully, and I want to run "python tools/converter.py run --config=mace/models/deeplab-v3-plus/deeplab-v3-plus-mobilenet-v2.yml -validate" but no responses. I have already plug a phone and open the developer mode.
I don't know the reason, please help me.

**Comment by nolanliou** [comment]
Nothing printed?

**Comment by raninbowlalala** [comment]
@nolanliou yes, nothing printed

**Comment by yejw5** [comment]
@raninbowlalala Can you try the latest code?

**Comment by yejw5** [comment]
Is this all output printed? Can you run as follows, and check devices logs:
```
# clear device logs
adb logcat -c
# run models
python tools/converter.py run --config=mace/models/deeplab-v3-plus/deeplab-v3-plus-mobilenet-v2.yml --round=100
# check device log
adb logcat
```

The normal logs look like:
```
adb logcat | grep mace_run
05-06 05:29:06.059 10954 10954 W linker  : "/data/local/tmp/mace_run/mace_run_dynamic" unused DT entry: type 0xf arg 0x1407
05-06 05:29:06.061 10954 10954 W linker  : "/data/local/tmp/mace_run/libmace.so" unused DT entry: type 0xf arg 0x1c4e
05-06 05:29:06.073 10954 10954 I MACE    : mace_run.cc:428 model name: preview_net
05-06 05:29:06.073 10954 10954 I MACE    : mace_run.cc:429 mace version: v0.8.1-138-g691331b-20180717
05-06 05:29:06.073 10954 10954 I MACE    : mace_run.cc:430 input node: input
05-06 05:29:06.073 10954 10954 I MACE    : mace_run.cc:431 input shape: 1,64,64,3
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:432 output node: softmax/Reshape_1
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:433 output shape: 1,64,64,2
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:434 input_file: /data/local/tmp/mace_run/model_input
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:435 output_file: /data/local/tmp/mace_run/model_out
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:436 model_data_file: /data/local/tmp/mace_run/preview_net.data
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:437 model_file: /data/local/tmp/mace_run/preview_net.pb
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:438 device: CPU
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:439 round: 1
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:440 restart_round: 1
05-06 05:29:06.074 10954 10954 I MACE    : mace_run.cc:441 gpu_perf_hint: 3
05-06 05:29:06.075 10954 10954 I MACE    : mace_run.cc:442 gpu_priority_hint: 3
05-06 05:29:06.075 10954 10954 I MACE    : mace_run.cc:443 omp_num_threads: -1
05-06 05:29:06.075 10954 10954 I MACE    : mace_run.cc:444 cpu_affinity_policy: 1
05-06 05:29:06.075 10954 10954 I MACE    : mace_run.cc:467 restart round 0
05-06 05:29:06.087 10954 10954 I MACE    : mace_run.cc:273 Total init latency: 10.161 ms
05-06 05:29:06.087 10954 10954 I MACE    : mace_run.cc:313 Warm up run
05-06 05:29:06.109 10954 10954 I MACE    : mace_run.cc:343 1st warm up run latency: 21.204 ms
05-06 05:29:06.109 10954 10954 I MACE    : mace_run.cc:350 Run model
05-06 05:29:06.125 10954 10954 I MACE    : mace_run.cc:392 Average latency: 16.244 ms
05-06 05:29:06.126 10954 10954 I MACE    : mace_run.cc:415 Write output file /data/local/tmp/mace_run/model_out_softmax_Reshape_1 with size 8192 done.
```

**Comment by raninbowlalala** [comment]
@yejw5 This works for me, now I can get CPU runtime, but there is error with GPU runtime:
```
I mace_run.cc:428 model name: deeplab_v3_plus_mobilenet_v2
I mace_run.cc:429 mace version: v0.8.1-140-gda931bf-20180717
I mace_run.cc:430 input node: sub_7
I mace_run.cc:431 input shape: 1,513,513,3
I mace_run.cc:432 output node: ResizeBilinear_2
I mace_run.cc:433 output shape: 1,65,65,21
I mace_run.cc:434 input_file: /data/local/tmp/mace_run/model_input
I mace_run.cc:435 output_file: /data/local/tmp/mace_run/model_out
I mace_run.cc:436 model_data_file: /data/local/tmp/mace_run/deeplab_v3_plus_mobilenet_v2.data
I mace_run.cc:437 model_file: /data/local/tmp/mace_run/deeplab_v3_plus_mobilenet_v2.pb
I mace_run.cc:438 device: GPU
I mace_run.cc:439 round: 10
I mace_run.cc:440 restart_round: 1
I mace_run.cc:441 gpu_perf_hint: 3
I mace_run.cc:442 gpu_priority_hint: 3
I mace_run.cc:443 omp_num_threads: -1
I mace_run.cc:444 cpu_affinity_policy: 1
I mace_run.cc:467 restart round 0
W opencl_runtime.cc:302 There is no precompiled OpenCL binary file in [/data/local/tmp/mace_run/deeplab-v3-plus-mobilenet-v2_compiled_opencl_kernel.SM-G950F.exynos5.bin]
I mace.cc:379 Create MaceEngine from model pb
I mace.cc:165 Creating MaceEngine, MACE version: v0.8.1-140-gda931bf-20180717
I mace.cc:233 Loading Model Data
I mace.cc:173 Initializing MaceEngine
W tuner.h:144 Read OpenCL tuned parameters file failed.
I file_storage.cc:40 File /data/local/tmp/mace_run/interior//mace_cl_compiled_program.bin does not exist
W opencl_runtime.cc:418 There is no precompiled OpenCL binary in all OpenCL binary paths
I mace_run.cc:273 Total init latency: 876.7 ms
I mace_run.cc:313 Warm up run
F helper.cc:339 Check failed: error == CL_SUCCESS error: CL_INVALID_WORK_GROUP_SIZE
Aborted (core dumped)
ERROR: [Mace Run] Mace run failed.

```

**Comment by yejw5** [comment]
@raninbowlalala Can you run with `python tools/converter.py run --config=mace/models/deeplab-v3-plus/deeplab-v3-plus-mobilenet-v2.yml --round=100 --vlog_level=5 > mace_run.log`. And upload the log file.

**Comment by raninbowlalala** [comment]
this is the last log in mace_run.log, do you need more log?
```
I mace_run.cc:313 Warm up run
I opencl_allocator.cc:47 Allocate OpenCL buffer: 3158092
I logging.h:104 clCreateBuffer started
I logging.h:110 clCreateBuffer latency: 3283 us
I logging.h:104 clRetainCommandQueue started
I logging.h:110 clRetainCommandQueue latency: 235 us
I logging.h:104 clEnqueueMapBuffer started
I logging.h:110 clEnqueueMapBuffer latency: 2516 us
I logging.h:104 clReleaseCommandQueue started
I logging.h:110 clReleaseCommandQueue latency: 172 us
I logging.h:104 clRetainCommandQueue started
I logging.h:110 clRetainCommandQueue latency: 264 us
I logging.h:104 clEnqueueUnmapMemObject started
I logging.h:110 clEnqueueUnmapMemObject latency: 263 us
I logging.h:104 clReleaseCommandQueue started
I logging.h:110 clReleaseCommandQueue latency: 217 us
I logging.h:104 Running net started
I logging.h:104 Running operator sub_7(BufferToImage), mem_id: [20000] started
I opencl_runtime.cc:483 cached OpenCL binary version is not same with current version
I logging.h:104 clCreateProgramWithSource started
I logging.h:110 clCreateProgramWithSource latency: 288 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 199 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 174 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 219 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 270 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 190 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 242 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 203 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 181 us
I logging.h:104 clBuildProgram started
I logging.h:110 clBuildProgram latency: 254202 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 153 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 220 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 194 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 199 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 179 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 179 us
I logging.h:104 clGetProgramBuildInfo started
I logging.h:110 clGetProgramBuildInfo latency: 164 us
I logging.h:104 clGetProgramBuildInfo started
I logging.h:110 clGetProgramBuildInfo latency: 167 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 171 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 130 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 157 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 171 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 179 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 173 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 147 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 165 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 4155 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 3798 us
I opencl_runtime.cc:605 Program from source: buffer_to_image -DCMD_DATA_TYPE=f -DDATA_TYPE=float -Din_out_buffer_to_image=i3g6CCkxG344HrLHuDI8ED
I logging.h:104 clRetainProgram started
I logging.h:110 clRetainProgram latency: 176 us
I logging.h:104 clCreateKernel started
I logging.h:110 clCreateKernel latency: 35838 us
I logging.h:104 clReleaseProgram started
I logging.h:110 clReleaseProgram latency: 152 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 193 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 246 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 171 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 162 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 186 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 201 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 147 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 189 us
I logging.h:104 clGetKernelWorkGroupInfo started
I logging.h:110 clGetKernelWorkGroupInfo latency: 179 us
I logging.h:104 clEnqueueNDRangeKernel started
I logging.h:110 clEnqueueNDRangeKernel latency: 244 us
I logging.h:104 clReleaseEvent started
I logging.h:110 clReleaseEvent latency: 168 us
I logging.h:104 clReleaseKernel started
I logging.h:110 clReleaseKernel latency: 192 us
I net.cc:126 Operator sub_7 has shape: [1, 513, 513, 3]
I logging.h:110 Running operator sub_7(BufferToImage), mem_id: [20000] latency: 317063 us
I logging.h:104 Running operator MobilenetV2/Conv/Relu6(Conv2D), mem_id: [20001] started
I opencl_runtime.cc:483 cached OpenCL binary version is not same with current version
I logging.h:104 clCreateProgramWithSource started
I logging.h:110 clCreateProgramWithSource latency: 186 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 185 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 154 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 181 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 161 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 170 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 171 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 141 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 157 us
I logging.h:104 clBuildProgram started
I logging.h:110 clBuildProgram latency: 59229 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 152 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 177 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 188 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 176 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 259 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 194 us
I logging.h:104 clGetProgramBuildInfo started
I logging.h:110 clGetProgramBuildInfo latency: 264 us
I logging.h:104 clGetProgramBuildInfo started
I logging.h:110 clGetProgramBuildInfo latency: 190 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 173 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 224 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 141 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 168 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 213 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 188 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 218 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 154 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 1109 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 998 us
I opencl_runtime.cc:605 Program from source: conv_2d_3x3 -DBIAS -DCMD_DATA_TYPE=f -DDATA_TYPE=float -DUSE_RELUX -Dconv_2d_3x3=c44CimylqKs
I logging.h:104 clRetainProgram started
I logging.h:110 clRetainProgram latency: 183 us
I logging.h:104 clCreateKernel started
I logging.h:110 clCreateKernel latency: 120226 us
I logging.h:104 clReleaseProgram started
I logging.h:110 clReleaseProgram latency: 150 us
I logging.h:104 clGetKernelWorkGroupInfo started
I logging.h:110 clGetKernelWorkGroupInfo latency: 179 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 175 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 188 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 181 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 216 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 201 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 186 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 190 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 276 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 156 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 139 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 157 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 165 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 160 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 164 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 156 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 136 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 180 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 163 us
I logging.h:104 clEnqueueNDRangeKernel started
I logging.h:110 clEnqueueNDRangeKernel latency: 1004 us
I logging.h:104 clReleaseEvent started
I logging.h:110 clReleaseEvent latency: 127 us
I net.cc:126 Operator MobilenetV2/Conv/Relu6 has shape: [1, 257, 257, 32]
I logging.h:110 Running operator MobilenetV2/Conv/Relu6(Conv2D), mem_id: [20001] latency: 202254 us
I logging.h:104 Running operator MobilenetV2/expanded_conv/depthwise/Relu6(DepthwiseConv2d), mem_id: [20002] started
I opencl_runtime.cc:483 cached OpenCL binary version is not same with current version
I logging.h:104 clCreateProgramWithSource started
I logging.h:110 clCreateProgramWithSource latency: 194 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 153 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 180 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 189 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 186 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 176 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 185 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 193 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 177 us
I logging.h:104 clBuildProgram started
I logging.h:110 clBuildProgram latency: 91899 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 149 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 386 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 199 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 194 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 180 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 187 us
I logging.h:104 clGetProgramBuildInfo started
I logging.h:110 clGetProgramBuildInfo latency: 147 us
I logging.h:104 clGetProgramBuildInfo started
I logging.h:110 clGetProgramBuildInfo latency: 161 us
I logging.h:104 clGetDeviceInfo started
I logging.h:110 clGetDeviceInfo latency: 158 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 169 us
I logging.h:104 clGetPlatformInfo started
I logging.h:110 clGetPlatformInfo latency: 157 us
I logging.h:104 clRetainDevice started
I logging.h:110 clRetainDevice latency: 180 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 213 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 177 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 179 us
I logging.h:104 clReleaseDevice started
I logging.h:110 clReleaseDevice latency: 173 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 1336 us
I logging.h:104 clGetProgramInfo started
I logging.h:110 clGetProgramInfo latency: 1278 us
I opencl_runtime.cc:605 Program from source: depthwise_conv2d -DBIAS -DCMD_DATA_TYPE=f -DDATA_TYPE=float -DSTRIDE=1 -DUSE_RELUX -Ddepthwise_conv2d_s1=du6A0F3D1n1DDMv7uNy
I logging.h:104 clRetainProgram started
I logging.h:110 clRetainProgram latency: 165 us
I logging.h:104 clCreateKernel started
I logging.h:110 clCreateKernel latency: 75840 us
I logging.h:104 clReleaseProgram started
I logging.h:110 clReleaseProgram latency: 149 us
I logging.h:104 clGetKernelWorkGroupInfo started
I logging.h:110 clGetKernelWorkGroupInfo latency: 174 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 156 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 179 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 190 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 206 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 183 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 260 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 183 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 183 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 180 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 170 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 190 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 172 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 257 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 208 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 193 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 171 us
I logging.h:104 clSetKernelArg started
I logging.h:110 clSetKernelArg latency: 207 us
I logging.h:104 clEnqueueNDRangeKernel started
I logging.h:110 clEnqueueNDRangeKernel latency: 196 us
F helper.cc:339 Check failed: error == CL_SUCCESS error: CL_INVALID_WORK_GROUP_SIZE
Aborted (core dumped)
[91mERROR: [Mace Run] Mace run failed.[0m
```

**Comment by yejw5** [comment]
@raninbowlalala We have found the root source. It is caused by the default local work size for some opencl op overflowing. It will be fixed soon. Thank you for your feedback.

**Comment by yejw5** [comment]
@raninbowlalala The problem is fixed in commit [6f59791d](https://github.com/XiaoMi/mace/commit/6f59791d099167c947afe777e8aec2419ef3eda1). Could you try it again.
